### DISCUSSION TITLE
About "Pull Shark" and "YOLO" badge.

### DISCUSSION THREAD

**Discussion by devcrafta** [community/community#18205] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello. **@Everyone**

I got the Pull Shark and YOLO badge yesterday.
Can someone please explain this badge?

**Comment by ale-lit** [comment]
Hi! 

This is new on GitHub. You can read more about it here:
https://github.com/orgs/github-community/discussions/18153
https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

---

<img src="https://github.githubassets.com/images/modules/profile/achievements/pull-shark-default.png" width="110" align="left">
<br>
<br>
<p valign="center">  Achievement "<b>Pull Shark</b>" given for generated and accepted pull request </p>
<br clear="all">

---

<img src="https://github.githubassets.com/images/modules/profile/achievements/yolo-default.png" width="110" align="left">
<br>
<br>
Achievement "<b>YOLO</b>" given for independent merging of branches in the project
<br clear="all">

---

<img src="https://github.githubassets.com/images/modules/profile/achievements/galaxy-brain-default.png" width="110" align="left">
<br>
<br>
Achievement "<b>Galaxy Brain</b>" given for a helpful answer to a question on GitHub Discussions

**Reply by devcrafta** [reply]
Thanks for help.

**Comment by dev-718** [comment, marked as answer]
Hi Astro.

Achievements celebrate specific events and actions that happen on GitHub. They will appear as small badges listed in the sidebar of your profile and are in Public Beta starting today. Check out the blog post to learn more.

https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

**Reply by devcrafta** [reply]
Thanks...

**Comment by devcrafta** [comment]
I guess gamification is a natural addition to any product at this point.
Only merged pull requests will count toward Pull Shark.

**Comment by ghost** [comment]
Hi there!
I suggest that you check this link.
https://github.com/Schweinepriester/github-profile-achievements.
Best regards.

**Reply by Casmei** [reply]
Thanks...
